.github/scripts/build-benchmark-artifact-remote.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Unless you have specific need otherwise, I'm going to suggest starting all bash scripts like:

```bash
#!/usr/bin/env bash

set -o errexit
set -o pipefail
set -o nounset
```

See https://kvz.io/bash-best-practices.html for reasons.